express-server/app.js

```diff
@@ -36,4 +36,19 @@ app.get('/apirecipeinfo/:recipeid', async (request, response) => {
     response.json(data);
 });
 
+// fetches standard popular recipes
+app.get('/apistdrecipes', async (request, response) => {
+    const params = {
+        apiKey: api_key,
+        number: 20,
+        sort: "random",
+        addRecipeInformation: true,
+        instructionsRequired: true,
+        fillIngredients: true
+    }
+    let recipes_url = `https://api.spoonacular.com/recipes/complexSearch`;
+    let { data } = await axios(recipes_url, { params })
+    response.json(data.results);
+});
+
```

Comment:
This doesn't seem related to error handling, it should probably be in it's own PR